mne/encoding/feature.py

```diff
@@ -0,0 +1,341 @@
+"""A collection of classes for transforming, preprocessing, and fitting."""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+import numpy as np
+from scipy import sparse
+from ..utils import warn, _reject_data_segments
+
+
+class DataDelayer(object):
```

Comment:
>if you wanted to create a different number of delays for different input features you could just iterate through the input columns using a custom window size each time, and then hstack the outputs.

Okay, then I think it's less convenient than it has to be - *from the perspective of somebody who often uses different time windows for different event types*. Seems it doesn't matter for straight forward STRF.